docs-ref-services/postgresql-mysql.md

```diff
@@ -54,46 +54,50 @@ Please see the [Python SDK authentication](https://docs.microsoft.com/python/azu
 
 #### Example
 
-Create a MySQL 5.7 Database resource and restrict access to a range of IP addresses using a firewall rule.
+Creates an Azure Database for MySQL server and restricts access to a range of IP addresses using a firewall rule.
 
 ```python
-
 from azure.mgmt.rdbms.mysql import MySQLManagementClient
-from azure.mgmt.rdbms.mysql.models import ServerForCreate, ServerPropertiesForDefaultCreate, ServerVersion
+from azure.mgmt.rdbms.mysql.models import *
 
 SUBSCRIPTION_ID = "YOUR_AZURE_SUBSCRIPTION_ID"
```

Comment:
It would be great if this sample could show the user what they need to do to prepare to execute this script.  i.e. Create a MySQL db, create admin user, get password,etc.